lib/DateTime/Format/Strptime/Types.pm

```diff
@@ -30,6 +30,7 @@ coerce(
 );
 
 object_isa_type( 'TimeZone', class => 'DateTime::TimeZone' );
+object_isa_type( 'DateTime', class => 'DateTime' );
```

Comment:
You can just write `object_isa_type('DateTime')` here, because the class name and type name are identical.